### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by isanex** [owncloud/ios-app#1279]
Hello,

I have no success executing the basic version. Every time I access to files the console return the same: 

'NSFileProviderManager: documentStorageURL is not set. Extension may be missing NSExtensionFileProviderDocumentGroup key, or current process may be missing necessary entitlements to access the specified container.'

Maybe I am missing something?
Thanks

### Expected behavior
Interact correctly with the files without the crash

### Actual behavior
Crashing every time with abort signal and NSFileProviderManager error

### Client
**iOS version: 14

**ownCloud app version: 12.0.3

**Device model: Simulator & iPhone 12

### Server configuration

All servers

**Comment by isanex** [comment]
![Captura de Pantalla 2023-09-20 a las 20 21 24](https://github.com/owncloud/ios-app/assets/138372130/2748c3cd-be44-435a-b626-249e5776923a)

Here the screenshot.

**Comment by michaelstingl** [comment]
Did you follow the instructions?
- https://github.com/owncloud/ios-app/blob/master/SETUP.md

Especially
https://github.com/owncloud/ios-app/blob/11074347386021d2bd222aee4efa14810cf93f70/SETUP.md?plain=1#L21-L23

**Comment by isanex** [comment]
Yes, all the steps. I've tried with all versions from 12, but same error with all of them.

![Captura de Pantalla 2023-09-22 a las 9 09 16](https://github.com/owncloud/ios-app/assets/138372130/21cdeb40-81e2-43f4-9dbe-84ac76b5922f)

**Comment by ChrAnd2** [comment]
Did you find any soloution for that @isanex ?
Im having the same problem :(

**Comment by felix-schwarz** [comment]
Closing for inactivity.
